### PR TITLE
Update getting started docs

### PR DESCRIPTION
**Pull request by FinHorsley**
### What are the main goals of this change?
- [x] Better readability (style, grammar, spelling...)
- [ ] Content correction (accuracy, wrong information...)
- [ ] New content

### Notes
This was an attempt to try and make it easier for people to get started with ReactiveUI; i removed and changed the layout of a lot of stuff here, so please reply with any feedback 🙂 

**Outline of major changes:** 
- Removed a lot of the top text (maybe too much? 😬) to make it really obvious what ReactiveUI is. Could the current text be moved into a _Fundementals of ReactiveUI_ doc for example?
- The [Compelling Example](https://reactiveui.net/docs/getting-started/#a-compelling-example) is great, but i think it cluttered the getting started page. I've linked to it (hopefully obviously enough) from the index.md
- Add some new links in the _Exploring ReactiveUI_ , such as [Pharmacist](https://github.com/reactiveui/Pharmacist)

### CODE REVIEW

**Comment by glennawatson** [comment]
Hey there. I will read through this over the next day or so.

File: input/docs/getting-started/compelling-example.md [hunk 1]
```diff
@@ -0,0 +1,467 @@
+Title: A Compelling Example
+Order: 10
+---
+
+Let's create a simple application demonstrating a number of ReactiveUI functionalities, without getting into too many under-the-hood details. We will create a WPF application, which will allow us to search through NuGet public repositories. The full code of the application is shown at the end of this chapter, and we will show relevant snippets as we go.
+
+### 1. Create the project
+
+In <a href="[https://visualstudio.microsoft.com/](https://visualstudio.microsoft.com/)">Visual Studio</a>, create a new WPF application (.NET 4.6.1 or above), use `ReactiveDemo` app name. Our view has been already created for us, the `MainWindow`, so we will proceed with creating our ViewModel.
```
Comment (glennawatson) [inline input/docs/getting-started/compelling-example.md:9]: Probably should update this to .net core 3.1 now

**Comment by glennawatson** [comment]
Overall good, sorry for the slow review.

**Comment by glennawatson** [comment]
I'll update to mention the .NET Core stuff, figured this PR has been sitting long enough. Thanks for the help.

File: input/docs/getting-started/compelling-example.md (hunk 1 above)
Comment (FinHorsley) [reply input/docs/getting-started/compelling-example.md:9]: Haha, things are out of date too quickly! 😅

**Comment by FinHorsley** [comment]
Thanks for the update 🙂 happy to make the 3.1 changes at some point

**Comment by FinHorsley** [comment]
Looks like the links in the ‘Explore ReactiveUI’ are all broken, I’ll try and get those fixed

**Pull request merged**